src/ztex/device_format.c

```diff
@@ -67,13 +69,16 @@ struct task_list *task_list;
 struct fmt_params *jtr_fmt_params;
 struct device_bitstream *jtr_bitstream;
 struct list_main *jtr_devices_allow;
+int jtr_verbosity;

```

Comment:
Global variables not used from outside of a translation unit should be declared `static`. I doubt all of these are used from elsewhere.